packages/mux-player/test/player.test.js

```diff
@@ -347,6 +347,88 @@ describe('<mux-player>', () => {
+      const playerEl = await fixture(`<mux-player
+        playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
+        stream-type="vod"
+      ></mux-player>`);
+
+      const mediaControllerEl = playerEl.shadowRoot.querySelector('media-controller');
+      const seekToLiveEl = playerEl.shadowRoot.querySelector('.mxp-seek-to-live-button');
+      assert.exists(mediaControllerEl);
+      assert.notExists(seekToLiveEl);
+    });
+
+    it('should have a seek to live button if the stream-type is live', async function () {
+      const playerEl = await fixture(`<mux-player
+        playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
+        stream-type="live"
+      ></mux-player>`);
+
+      const mediaControllerEl = playerEl.shadowRoot.querySelector('media-controller');
+      const seekToLiveEl = playerEl.shadowRoot.querySelector('.mxp-seek-to-live-button');
+      assert.exists(mediaControllerEl);
+      assert.exists(seekToLiveEl);
+    });
+
+    it('should have a seek to live button if the stream-type is ll-live', async function () {
+      const playerEl = await fixture(`<mux-player
+        playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
+        stream-type="ll-live"
+      ></mux-player>`);
+
+      const mediaControllerEl = playerEl.shadowRoot.querySelector('media-controller');
+      const seekToLiveEl = playerEl.shadowRoot.querySelector('.mxp-seek-to-live-button');
+      assert.exists(mediaControllerEl);
+      assert.exists(seekToLiveEl);
+    });
+
+    it('should should seek to live when seek to live button pressed', async function () {
+      this.timeout(12000);
+      const playerEl = await fixture(`<mux-player
+        playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
+        muted
+        stream-type="ll-live"
+      ></mux-player>`);
+
+      const seekToLiveEl = playerEl.shadowRoot.querySelector('.mxp-seek-to-live-button');
+      // NOTE: Need try catch due to bug in play+autoplay behavior (CJP)
+      try {
+        await playerEl.play();
+      } catch (_e) {}
+      await waitUntil(() => !playerEl.paused, 'play() failed');
+      await waitUntil(() => playerEl.inLiveWindow, 'playback did not start inLiveWindow');
+      playerEl.pause();
+      await waitUntil(() => !playerEl.inLiveWindow, 'still inLiveWindow after long pause', { timeout: 7500 });
+      seekToLiveEl.click();
+      await waitUntil(() => playerEl.inLiveWindow, 'clicking seek to live did not seek to live window');
+      return Promise.resolve();
+    });
+
+    it('should should seek to live when play button is pressed', async function () {
+      this.timeout(12000);
+      const playerEl = await fixture(`<mux-player
+        playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
+        muted
+        stream-type="ll-live"
+      ></mux-player>`);
+
+      const mcPlayEl = playerEl.shadowRoot.querySelector('media-play-button');
+      // NOTE: Need try catch due to bug in play+autoplay behavior (CJP)
+      try {
```

Comment:
this needs to be ticketed and is related to some of the bad autoplay behavior I noticed during previous efforts.